pkg/restapi/vc/operation/operations.go

```diff
@@ -866,6 +875,77 @@ func (o *Operation) generateKeypairHandler(rw http.ResponseWriter, req *http.Req
 	})
 }
 
+func (o *Operation) credentialVerificationsHandler(rw http.ResponseWriter, req *http.Request) {
+	// get the request
+	verificationReq := CredentialVerificationsRequest{}
+
+	err := json.NewDecoder(req.Body).Decode(&verificationReq)
+	if err != nil {
+		o.writeErrorResponse(rw, http.StatusBadRequest, fmt.Sprintf(invalidRequestErrMsg+": %s", err.Error()))
+
+		return
+	}
+
+	// TODO What are the checks to perform if options are not passed ? For now, made options and checks mandatory.
```

Comment:
Default for empty checks seems to be all checks.

> A default list is also expected to encapsulate all available checks

Reply:
@rolsonquadras will update in followup PR.

Reply:
Created an issue https://github.com/trustbloc/edge-service/issues/133